SteakBot.Core/EventHandlers/CustomMessageHandlers/CommandMessageHandlers/CustomCommandMessageHandler.cs

```diff
@@ -54,16 +50,9 @@ protected override bool InvokeInner(SocketUserMessage message)
 
         #region Private methods
 
-        private static IList<MemeCommand> LoadCommands()
+        public void ReloadCommandNames()
```

Comment:
This isn't used anywhere outside of this class so can go back to being `private`.